cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -185,6 +190,10 @@ func run(ctx context.Context, target, assetType, optJSON string, state checkstat
 	}
 
 	trivyArgs := []string{}
+
+	// Increase the default (5m) trivy command timeout.
+	trivyArgs = append(trivyArgs, []string{"--timeout", trivyTimeout}...)
+
```

Comment:
Why not? 
```go
trivyArgs = append(trivyArgs, "--timeout", trivyTimeout)
```

Reply:
As I mentioned to Roi in previous comments:

I agree with the suggestion, but I was just trying to follow the same constructions that were already made in the check.